File: csla20vb/Csla/FilterProvider.vb
Imports System

''' <summary>
''' Defines the method signature for a filter
''' provider method used by FilteredBindingList.
''' </summary>
''' <param name="item">The object to be filtered.</param>
''' <param name="filter">The filter criteria.</param>
''' <returns><see langword="true"/> if the item matches the filter.</returns>
Public Delegate Function FilterProvider(ByVal item As Object, ByVal filter As Object) As Boolean

File: cslacs/Csla/FilterProvider.cs
using System;

namespace Csla
{
  /// <summary>
  /// Defines the method signature for a filter
  /// provider method used by FilteredBindingList.
  /// </summary>
  /// <param name="item">The object to be filtered.</param>
  /// <param name="filter">The filter criteria.</param>
  /// <returns><see langword="true"/> if the item matches the filter.</returns>
  public delegate bool FilterProvider(object item, object filter);
}
